### DISCUSSION TITLE
ENABLE MY COPILOT FEATURES !!!!!

### DISCUSSION THREAD

**Discussion by ShiningYangYXN** [community/community#17307] [Copilot Conversations] [answered]
# Please enable my copilot features!

![Screenshot_20220526-205315](https://user-images.githubusercontent.com/97453940/170491483-3f38617e-baa5-4fb1-94b2-9e1d404d6c83.png)

I joined the waitlist many days ago, but there is no response yet. How can I get the access more quickly?

**Comment by davecheney** [comment, marked as answer]
I'm sorry that you have not been given access yet. Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060
